operator/src/main/resources/scripts/livenessProbe.sh

```diff
@@ -91,6 +91,16 @@ if [ -f ${STATEFILE} ] && [ `grep -c "FAILED_NOT_RESTARTABLE" ${STATEFILE}` -eq
   exit $RETVAL
 fi
 
+if [ -x ${LIVENESS_PROBE_CUSTOM_SCRIPT} ]; then
```

Comment:
If the plug-in script is to support WL_HOME, ORACLE_HOME, and MW_HOME it needs a call 'exportInstallHomes' (just like the the script already calls exportEffectiveDomainHome ^^^ which takes care of DOMAIN_HOME).